packages/protocol/contracts/thirdparty/automata-attestation/interfaces/IAttestation.sol

```diff
@@ -0,0 +1,11 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { V3Struct } from "../lib/QuoteV3Auth/V3Struct.sol";
+
+interface IAttestation {
+    function verifyAttestation(bytes calldata data) external returns (bool);
```

Comment:
I think we can remove this one from the interface as it costs more gas and actually never be called now. 
Even the code can be also removed, however, let's keep the code for merely reference now, and clean it up after we  have enough tests.

Reply:
Maybe best to keep original interface ? Does not harm.

fyi: also added a thirdparty reference readme.md in contracts/thirdparty, please check if that is correct or i should remove you repo's link.

Reply:
No harm, I will create another PR to move sth (will be) deprecated to test sol. Feels like a advantage of foundry as you can alway keep the last version in test because it is still sol.